src/UKHO.S100PermitService.Common/Helpers/FileSystemHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Helpers
+{
+    [ExcludeFromCodeCoverage]
+    public class FileSystemHelper : IFileSystemHelper
```

Comment:
Please don't use 'helper' classes, they really break the Single Responsibility principle. Could you combine these two classes (FileSystemHelperXmlHelper) into something like a PermitReaderWriter? That makes its intent and scope clear. 

This could then be injected into the IPermitService, which will remove a lot of logic from the controller?

Reply:
Implemented the PermitReaderWriter and injected IPermitService to it